lib/anidb/category_importer.rb

```diff
@@ -0,0 +1,45 @@
+class AnidbCategoryImport
+  class ImportFile
```

Comment:
I think we want to keep the class names matching the filename, so `anidb/category_importer.rb` would be `AniDB::CategoryImporter` and `AnidbCategoryImport::ImportFile` would be `anidb_category_import/import_file.rb` — though honestly you can just keep `ImportFile` inline, but probably want to rename the outermost class here to match the file.